src/modules/grubcfg/main.py

```diff
@@ -90,6 +90,9 @@ def modify_grub_default(partitions, root_mount_point, distributor):
     swap_outer_uuid = ""
     swap_outer_mappername = None
     no_save_default = False
+    unencrypted_separate_boot = False
+    if any(partition["mountPoint"] == "/boot" and "luksMapperName" not in partition for partition in partitions):
+        unencrypted_separate_boot = True
```

Comment:
Since `any()` returns a boolean, you can write this shorter as
```
unencrypted_separate_boot = any(partition["mountPoint"] == "/boot" and "luksMapperName" not in partition for partition in partitions)
```
(maybe rename *partition* to *p* to make the line shorter, which is ok since *p* is only in the scope of this list comprehension)

Reply:
That's a great idea, it makes it look quite a bit prettier!